webtau-cli-groovy/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.twosigma.webtau</groupId>
         <artifactId>webtau-parent</artifactId>
-        <version>1.9-SNAPSHOT</version>
+        <version>1.10-SNAPSHOT</version>
```

Comment:
@MykolaGolubyev any ideas why the release:prepare might not have updated these two modules?

Reply:
My guess is I did branch and didn’t merge later.